src/App.tsx

```diff
@@ -15,36 +15,104 @@
   'Garlic',
 ];
 
+enum SortType {
+  DEFAULT = 'default',
+  ALPHABET = 'alphabet',
+  LENGTH = 'length',
+}
+
+const preparedGoods = [...goodsFromServer];
+
+const sortGoods = (order: SortType, isReversed: boolean): string[] => {
```

Comment:
The function name 'sortGoods' should start with a verb that describes its action more precisely. Consider renaming it to something like 'getSortedGoods' to better convey its purpose. See checklist item 4.